src/DoctrineORMModule/Module.php

```diff
@@ -149,4 +151,33 @@ public function initializeConsole(EventInterface $event)
         $helperSet->set(new ConnectionHelper($entityManager->getConnection()), 'db');
         $helperSet->set(new EntityManagerHelper($entityManager), 'em');
     }
+
+    /**
+     * Expected to return \Zend\ServiceManager\Config object or array to
+     * seed such an object.
+     *
+     * @return array|\Zend\ServiceManager\Config
+     */
+    public function getRouteConfig()
```

Comment:
as above, we should keep it in `module.config.php` (this way it is always done in zend/apigility modules)